data/wanderer/wanderers start.txt

```diff
@@ -1312,6 +1318,8 @@ mission "Wanderers: Alpha Surveillance E"
 			choice
 				`	"Are you sure we can trust them? The Alphas were in a Navy ship, after all."`
 				`	"We could go to Admiral Danforth. He's a Navy officer who I trust to be on the right side."`
+					to display
+						not "skip FW"
 					goto danforth
 			`	"Can you think of any other option?" he says.`
 			`	"There's a branch of the Navy called the 'Oathkeepers,'" you tell him, "led by Admiral Danforth. If there is anyone in the Navy whom we can trust, it's them."`
```

Comment:
```suggestion
			`	As you think about the Navy, you recall a group called the Oathkeepers that you heard about before you moved back to the Hai. "There's a branch of the Navy called the 'Oathkeepers,'" you tell him, "led by Admiral Danforth. If there is anyone in the Navy whom we can trust, it's them."`
```

I think you should tell the player where they learned about the Oathkeepers. I'm not sure about my wording, though.

Reply:
You're right, this part escaped me because I thought he was speaking for some reason
This before you moved to hai space should only show if you went ith the hai start and also you grew up in Hai space here

So I think the best option would be for him to suggest it? Or just to say you heard about it from humans coming in hai space

Reply:
From what you tell Danforth later, it seems you know about them.

Reply:
```suggestion
			`	As you consider the prospect of speaking to the Navy, you recall a Navy group, the Oathkeepers. They were led by the honorable Admiral Danforth. "There's a branch of the Navy called the 'Oathkeepers,'" you tell him, "led by Admiral Danforth. If there is anyone in the Navy whom we can trust, it's them."`
```

Reply:
I'll need to take a look at that too then, but I think its safe to assume you learned about them because they are taking care of the pirates "next door"? 

How about saying that a merchant that got saved by them speaked to you?

Reply:
That sounds good (both of those)

Reply:
I just noticed I already added a little smth at the end of the mission, but this cant hurt
does it look good?

Reply:
What you wrote looks good!